db-scripts/insert-topic/README.md

```diff
@@ -0,0 +1,64 @@
+# insert-topics
+
+This utility inserts topics and their subtopics into a MongoDB collection. Each new topic can be added under a different specified parent topic identified by its `slug`.
+
+### How to run the utility
```

Comment:
Should be structuring headers using # correctly - this should be a ## for example.